sale_order_import_invoice2data/README.rst

```diff
@@ -0,0 +1,100 @@
+===================================
+Sale Order Import Saleorder2data
+===================================
+
+This module is an extension of the module *sale_order_import*: it adds support for regular PDF Sale Order i.e. PDF Sale Order that don't have an embedded XML file. It uses the `invoice2data library
+<https://github.com/m3nu/invoice2data>`_ which takes care of extracting the text of the PDF Sale Order, find an existing invoice template and execute the invoice template to extract the useful
+information from the invoice.
+
+To know the full story behind the development of this module, read this `blog post <http://www.akretion.com/blog/akretions-christmas-present-for-the-odoo-community>`_.
+
+Installation
+============
+
+This module requires the Python library *invoice2data* available on `Github <https://github.com/m3nu/invoice2data>`_.
+
+To install the right version of the library, run:
+
+.. code::
+
+  sudo pip install git+https://github.com/m3nu/invoice2data.git
+
+In order to use a recent version of invoice2data on Odoo v8, you need an Odoo v8 dated after January 29th 2016, so that it contains `this fix <https://github.com/odoo/odoo/commit/edeb5a8c0fb5c837364f1d92db731f89824bb28a>`_ which fixes `this bug <https://github.com/odoo/odoo/issues/10670>`_.
+
+The invoice2data library requires the latest version of the *pdftotext* utility, which is not yet packaged in Debian/Ubuntu. So you should download it from `the FTP server of Foolabs <ftp://ftp.foolabs.com/pub/xpdf/xpdfbin-linux-3.04.tar.gz>`_ then uncompress it and copy the file *bin64/pdftotext* to */usr/local/bin*.
```

Comment:
debian sid and ubuntu 16.04 have this already, so better point to the respective backports repositories